llvm/lib/Transforms/Scalar/IR2Vec-LOF/custom_loop_distribution/Readme.md

```diff
@@ -0,0 +1,109 @@
+
+# Loop Distribution
+
+This directory contains the Python model related files for training and inferencing Loop Distribution.
+
+
+
+This repo contains the source code and relevant information described in the [paper](https://ieeexplore.ieee.org/abstract/document/10026979)
+Please see [here](https://compilers.cse.iith.ac.in/publications/rl_loop_distribution/) for more details.
+
+> Reinforcement Learning assisted Loop Distribution for Locality and Vectorization, Shalini Jain, S. VenkataKeerthy, Rohit Aggarwal, Tharun Kumar Dangeti, Dibyendu Das, Ramakrishna Upadrasta
+
+
+## Environment Setup
+
+
+Setup the environment using the `ml-llvm-project/model/LoopDistribution/src/LOF_original_env.yml` file with the following command
```

Comment:
Clean this up. Directly say conda env create -f  file name.

Reply:
Done.